### DISCUSSION TITLE
Loading a beatmap bypasses search filter.

### DISCUSSION THREAD

**Discussion by rvnexe** [ppy/osu#23650] [Q&A]
### Type

Game behaviour

### Bug description

It's understandable in the video:

### Screenshots or videos

https://github.com/ppy/osu/assets/89677246/dc8830f0-69e7-4f44-89c5-c16f73a8230d

### Version

2023.513.0-lazer

### Logs

[updater.log](https://github.com/ppy/osu/files/11553180/updater.log)
[database.log](https://github.com/ppy/osu/files/11553183/database.log)
[input.log](https://github.com/ppy/osu/files/11553184/input.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/11553185/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/11553186/network.log)
[performance.log](https://github.com/ppy/osu/files/11553188/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11553189/runtime.log)

**Comment by peppy** [comment]
This is intended. How would you expect this to behave instead?

**Reply by rvnexe** [reply]
Shouldn't it clear the search and show the beatmap in the full list?

**Reply by integerrr** [reply]
That wouldn't work so well if you were to import a bunch of beatmaps and need to use the search function in the meanwhile

**Reply by rvnexe** [reply]
@integerrr Actually, The list will reset if you load another beatmap! Look at this:
https://github.com/ppy/osu/assets/89677246/c06d213f-0547-4bce-afbf-bfe0e69771cd
